templates/core/amp/layout/amp-iframe.html

```diff
@@ -25,7 +25,7 @@
 
     {{#block "page"}} {{/block}}
 
-    <script src="{{cdn '/assets/js/bundle.js'}}"></script>
+    <script src="{{cdn 'assets/dist/theme-bundle.main.js'}}"></script>
```

Comment:
Sholud we add `/` before path to bundle as it was before?

Reply:
regarding path: I checked on [Cornerstone](https://github.com/bigcommerce/cornerstone/blob/master/templates/layout/amp-iframe.html#L32), and in helpers [docs](https://developer.bigcommerce.com/stencil-docs/reference-docs/handlebars-helpers-reference#cdn) (see example with assets for cdn-helper) and in both cases it's being used without slash. I also doesn't notice any troubles in bundle test.

Reply:
in that case I believe its fine